src/indicators/idmc/update_idmc.R

```diff
@@ -88,7 +88,7 @@ df_idmc_flags <- flagging$generate_alerts(
 #### COMPARE WITH EXISTING FILES ####
 #####################################
 
-df_idmc_flags_prev <- gs$read_gs_file("flags_idmc") |>
+df_idmc_flags_prev <- cs$read_gcs_file("output/flags/idmc.parquet") |>
```

Comment:
need to add
```
box::use(cs = ../../utils/cloud_storage)
```

think you want to get rid of  google_sheets call in all scripts (here & cholera)

```
box::use(gs = ../../utils/google_sheets)
```